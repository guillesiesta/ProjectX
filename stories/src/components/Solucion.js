import React, {Component} from 'react';

export default class Solucion extends Component {
render(){

      return(
          <div className="box-header with-border">
            <div className="row">

              <li className="list-group-item">Soluci√≥n: <strong>{this.props.solucion}</strong> y Puntuacion: <strong>{this.props.puntuacion}</strong></li>
            </div>
          </div>
      )
  }
}
