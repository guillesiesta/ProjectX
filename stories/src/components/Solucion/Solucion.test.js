import React, {Component} from 'react';
import { shallow } from 'enzyme';

import Solucion from './Solucion';

import "isomorphic-fetch"

describe('Solucion', ()=> {
  test('Rendering the simple component', ()=>{

    const wrapper = shallow(
      <Solucion />
    );

    expect(wrapper).toMatchSnapshot();
  })
})
