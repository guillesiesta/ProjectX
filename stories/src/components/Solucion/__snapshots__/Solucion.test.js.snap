// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Solucion Rendering the simple component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Solucion />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="row"
      >
        <li
          className="list-group-item"
        >
          Solución: 
          <strong />
           y Puntuacion: 
          <strong>
             %
          </strong>
        </li>
      </div>,
      "className": "box-header with-border",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <li
          className="list-group-item"
        >
          Solución: 
          <strong />
           y Puntuacion: 
          <strong>
             %
          </strong>
        </li>,
        "className": "row",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Solución: ",
            <strong />,
            " y Puntuacion: ",
            <strong>
               %
            </strong>,
          ],
          "className": "list-group-item",
        },
        "ref": null,
        "rendered": Array [
          "Solución: ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "strong",
          },
          " y Puntuacion: ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                undefined,
                " %",
              ],
            },
            "ref": null,
            "rendered": Array [
              undefined,
              " %",
            ],
            "type": "strong",
          },
        ],
        "type": "li",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
        >
          <li
            className="list-group-item"
          >
            Solución: 
            <strong />
             y Puntuacion: 
            <strong>
               %
            </strong>
          </li>
        </div>,
        "className": "box-header with-border",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <li
            className="list-group-item"
          >
            Solución: 
            <strong />
             y Puntuacion: 
            <strong>
               %
            </strong>
          </li>,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Solución: ",
              <strong />,
              " y Puntuacion: ",
              <strong>
                 %
              </strong>,
            ],
            "className": "list-group-item",
          },
          "ref": null,
          "rendered": Array [
            "Solución: ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "strong",
            },
            " y Puntuacion: ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  undefined,
                  " %",
                ],
              },
              "ref": null,
              "rendered": Array [
                undefined,
                " %",
              ],
              "type": "strong",
            },
          ],
          "type": "li",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
